src/sentry/api/endpoints/group_notes.py

```diff
@@ -57,14 +59,30 @@ def post(self, request, group):
             reason=GroupSubscriptionReason.comment,
         )
 
-        if mentions:
-            users = User.objects.filter(id__in=mentions)
-            for user in users:
+        mentions = data.pop('mentions', [])
+
+        actors = Actor.resolve_many(mentions)
+        actorMentions = seperate_resolved_actors(actors)
+
+        subscribed_user_ids = set()
+
+        for user in actorMentions.get('users', []):
+            GroupSubscription.objects.subscribe(
+                group=group,
+                user=user,
+                reason=GroupSubscriptionReason.mentioned,
+            )
+            subscribed_user_ids.add(user.id)
+
+        for team in actorMentions.get('teams', []):
+            for user in [member.user for member in team.member_set
```

Comment:
you're gonna want to use `select_related` here to avoid running a query when accessing the user for every member. I think actually since `member_set` returns a queryset, you should be able to just chain  `team.member_set.select_related('user')`